test/e2e/workflow/deployment_test.go

```diff
@@ -165,6 +165,7 @@ var _ = Describe("NetworkAddonsConfig", func() {
 		//2300
 		It("should be able to remove the config and create it again", func() {
 			DeleteConfig()
+			CheckComponentsRemoval(components)
```

Comment:
This is adding a regression to our tests. Please document it with TODO and a linked issue. We need to drop this once we resolve it.

Reply:
done